src/pages/infopages/PrivacyPolicy.tsx

```diff
@@ -1,26 +1,64 @@
-import React, { FC } from 'react';
+import React from 'react';
 import styled from 'styled-components';
-import { useTranslation } from 'react-i18next';
-import Heading from '../../components/Heading';
+import { Trans, useTranslation } from 'react-i18next';
+import { Table, TableBody, TableCell, TableHead, TableRow, Typography } from '@material-ui/core';
 
-import Card from '../../components/Card';
-import { StyledInformationWrapper, StyledNormalTextPreWrapped } from '../../components/styled/Wrappers';
+import { PageHeader } from '../../components/PageHeader';
 
-const StyledHeading = styled(Heading)`
-  text-align: center;
-  padding-bottom: 1rem;
+const StyledPrivacyContainer = styled.div`
+  display: block;
 `;
 
-const PrivacyPolicy: FC = () => {
-  const { t } = useTranslation('infopages');
+const PrivacyPolicy = () => {
+  const { t } = useTranslation('privacy');
 
   return (
-    <StyledInformationWrapper data-testid="privacy-policy" aria-label={t('privacy_policy.heading')}>
-      <Card>
-        <StyledHeading>{t('privacy_policy.heading')}</StyledHeading>
-        <StyledNormalTextPreWrapped>{t('privacy_policy.description')}</StyledNormalTextPreWrapped>
-      </Card>
-    </StyledInformationWrapper>
+    <>
+      <PageHeader>{t('privacy_statement')}</PageHeader>
+
+      <StyledPrivacyContainer>
+        <Typography variant="h2">{t('about.heading')}</Typography>
+        <Typography>{t('about.description')}</Typography>
+
+        <Typography variant="h2">{t('what_are_personal_data.heading')}</Typography>
+        <Typography>
+          <Trans t={t} i18nKey="what_are_personal_data.description">
+            <a target="_blank" rel="noreferrer" href="https://lovdata.no/lov/2018-06-15-38/§2">
+              (i18n content: Personal Data Act Section 2)
+            </a>
+            <a target="_blank" rel="noreferrer" href="https://lovdata.no/lov/2018-06-15-38/gdpr/a5">
+              (i18n content: GDPR Article 5)
+            </a>
+          </Trans>
+        </Typography>
+
+        <Typography variant="h2">{t('service_in_brief.heading')}</Typography>
+        <Typography>{t('service_in_brief.description0')}</Typography>
+        <Typography>{t('service_in_brief.description1')}</Typography>
+        <Typography>{t('service_in_brief.description2')}</Typography>
+
+        <Typography variant="h2">{t('purpose.heading')}</Typography>
+        <Typography>{t('purpose.description')}</Typography>
+        <Table>
+          <TableHead>
+            <TableRow>
+              <TableCell>{t('purpose.table.header.col0')}</TableCell>
+              <TableCell>{t('common:description')}</TableCell>
```

Comment:
Litt forvirrende at det er col0 og col2, men ikke col1. Har vel ikke så mye å si om vi definerer beskrivelse to ganger? Enn om vi endrer på headeren på description taben? Det skal jo ikke nødvendigvis være samme tekst som her

Reply:
jepp, kan legge til col1 også